packages/server/src/jsRunner/vm/isolated-vm.ts

```diff
@@ -98,6 +98,20 @@ export class IsolatedVM implements VM {
     return this
   }
 
+  withSnippets(snippets?: Snippet[]) {
```

Comment:
Are we ever loading anything other than the full gamut of snippets? If not, I'd personally have the code to fetch them from the app metadata in here and remove adding the snippets to the request context. Or am I missing something?

Reply:
I would love to have them in here, and did initially, but there is a reason we can't - we can't have an async handlebars helper - so we can't do anything async here no matter what. Fetching the snippets from app metadata requires an async call, unless you can think of a clever way to get around that.

Reply:
Blergh, that's a shame.